src/server/index.js

```diff
@@ -0,0 +1,42 @@
+import * as Server from '@ucanto/server'
+import * as CAR from '@ucanto/transport/car'
+import { ok, error } from '@ucanto/core'
+import { DIDResolutionError } from '@ucanto/validator'
+
+/**
+ * Creates a UCAN server.
+ *
+ * @template T
+ * @param {import('../middleware/withUcanInvocationHandler.types.js').Context} ctx
+ * @param {import('./api.types.js').Service<T>} service
+ */
+export function createServer (ctx, service) {
+  return Server.create({
+    id: ctx.gatewaySigner,
+    codec: CAR.inbound,
+    service,
+    catch: err => console.error(err),
+    // TODO: wire into revocations
+    validateAuthorization: () => ({ ok: {} }),
+    // @ts-expect-error - The type is not defined in the ucan package, but it supports the method.
+    resolveDIDKey: (did) => resolveDIDKey(did, ctx)
```

Comment:
**question:** When do we need to resolve the Gateway's DID? The Gateway is the audience of these delegations, not the issuer. It shouldn't have signed anything that needs verifying. Right? Or am I missing something?

Reply:
Good catch. That's right. We only need it to resolve the DID keys when we want to validate the issuer, and the issuer is using a `did:web`. I just removed that.